src/backend/distributed/planner/multi_explain.c

```diff
@@ -710,3 +715,73 @@ ExplainYAMLLineStarting(ExplainState *es)
 		appendStringInfoSpaces(es->str, es->indent * 2);
 	}
 }
+
+
+/*
+ * CoordinatorInsertSelectExplainScan is a custom scan explain callback function
+ * which is used to print explain information of a Citus plan for an INSERT INTO
+ * distributed_table SELECT ... query that is evaluated on the coordinator.
+ */
+void
+CoordinatorInsertSelectExplainScan(CustomScanState *node, List *ancestors,
```

Comment:
The bottom of this file is all copy-pasted functions from `explain.c`, so this function doesn't belong here (and putting it here omits it from `citus_indent` consideration).